mappings/4de8f6ef521125e09d898267368ebeebf2287a562620daea3d853163546563686e6f6372616379.json

```diff
@@ -0,0 +1,34 @@
+{
+  "subject": "4de8f6ef521125e09d898267368ebeebf2287a562620daea3d853163546563686e6f6372616379",
+  "name": {
+    "sequenceNumber": 1,
+    "value": "Technocracy",
+    "signatures": [
+      {
+        "signature": "55845c63cf510c7d60eaf905d8165469cf5d0553a1acaf7eb9087d18a7831b2fd249387cb9ed40dfbb75e9419a1bdc6a4955a03edfe06ed964701d0acb00490f",
+        "publicKey": "f263a15d8e7751bef48998c81eb9eaec6ccc122d90514eff061d12392654df61"
+      }
+    ]
+  },
+  "decimals": {
+    "sequenceNumber": 1,
+    "value": 0,
+    "signatures": [
+      {
+        "signature": "1ebe33c814e3e987093026da240dac44b7a22060553e938f96c857aa3193359bf703f8c9b1c085289301f95d950a55055c56c2061dfd5ddad21d3b8f061b980b",
+        "publicKey": "f263a15d8e7751bef48998c81eb9eaec6ccc122d90514eff061d12392654df61"
+      }
+    ]
+  },
+  "policy": "820182018282051a03e4ca068200581cc92831d2b9023de89709dcde75baa266012e436b876818e652149ee0",
+  "description": {
+    "sequenceNumber": 1,
+    "value": "",
```

Comment:
The description should not be left blank, thank you for providing a description for the token!